projects/igniteui-angular/src/lib/directives/toggle/toggle.directive.ts

```diff
@@ -18,6 +18,14 @@ import { IgxOverlayService } from '../../services/overlay/overlay';
 import { OverlaySettings, OverlayEventArgs, ConnectedPositioningStrategy, AbsoluteScrollStrategy } from '../../services';
 import { filter, take } from 'rxjs/operators';
 import { Subscription, OperatorFunction } from 'rxjs';
+import { OverlayCancelableEventArgs } from '../../services/overlay/utilities';
+
+export interface CancelableEventArgs {
```

Comment:
You could place this interface on a common place so it could be reused.

Reply:
Moved to core/utils